storage/filesystem/littlefsv2/source/LittleFileSystem2.cpp

```diff
@@ -182,7 +182,8 @@ int LittleFileSystem2::mount(BlockDevice *bd)
     _config.sync            = lfs2_bd_sync;
     _config.read_size       = bd->get_read_size();
     _config.prog_size       = bd->get_program_size();
-    _config.block_size      = lfs2_max(_config.block_size, (lfs2_size_t)bd->get_erase_size());
+    _config.block_size      = lfs2_max(_config.block_size, lfs2_max((lfs2_size_t)bd->get_erase_size(0),
+                                                                    (lfs2_size_t)bd->get_erase_size(bd->size())));
```

Comment:
```suggestion
                                                                    (lfs2_size_t)bd->get_erase_size(bd->size() - 1)));
```
because virtual address ranges from 0 to size - 1....